plugins/cache-portals-on-map.js

```diff
@@ -29,38 +29,37 @@ window.plugin.cachePortalDetailsOnMap.entityInject = function(data) {
   }
   data.callback(ents);
 };
-/****************************************************************************************/
-/* extend portal's GUID lookup to search in cached portals as well                      */
 
-window.plugin.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
-  let guid = window.plugin.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+// extend portal's GUID lookup to search in cached portals as well
+window.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
+  let guid = window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+
   // PLUGIN: cache > cache
   if (guid == null) {
     const cache = window.plugin.cachePortalDetailsOnMap;
     if (cache && cache.cache) {
       for (let g in cache.cache) {
         let p = cache.cache[g];
         if (latE6 == p.ent[2][2] && lngE6 == p.ent[2][3]) {
-          console.log("findPortalGuidByCacheE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
+//        console.log("newFindPortalGuidByPositionE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
           guid = g;
           break;
         }
       }
     }
   }
   return guid;
-}
+};
+
+window.cachePortalDetailsOnMap.searchInit = function () {
+    window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old = window.findPortalGuidByPositionE6;
+    window.findPortalGuidByPositionE6 = window.cachePortalDetailsOnMap.findPortalGuidByCacheE6;
```

Comment:
Such hijacking is what we should avoid, especially in official plugins.

If you still need that badly - consider providing such possibility in some clean way (hooks, etc)

Reply:
Hooks cause all registered functions to execute. 
The idea here is to find a guid. The processing should stop as soon as ANY of the additional plugins was successful.

CachePortalsOnMap used to replace the core function, only extending it.
Offle-Plugin did the same but there was no way to get the benefit of both.
If the core function would change the plugins would both need to adjust.

If there is a better way, which does not cause useless CPU usage please commit.

Reply:
> The idea here is to find a guid. The processing should stop as soon as ANY of the additional plugins was successful.

Whatever. The API should be clean.
So even in this case I prefer hook instead of hijacking, difference in CPU will be miserable anyway.

But if you care than we can extend hooks api here.